<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>1271</width>
      <height>1348</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextEdit" name="img_label_2">
           <property name="minimumSize">
            <size>
             <width>365</width>
             <height>365</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>365</width>
             <height>365</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="img_label_3">
           <property name="minimumSize">
            <size>
             <width>730</width>
             <height>730</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>731</width>
             <height>730</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="plotWidget1" native="true">
       <property name="minimumSize">
        <size>
         <width>1260</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>831</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton1">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>选择文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txt_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
